frontend/src/services/api.js

```diff
@@ -0,0 +1,15 @@
+import axios from 'axios';
+const baseUrl = import.meta.env.VITE_BASE_URL;
+
+
+export const getCourses = async() => {
+    try{
```

Comment:
Do you have prettier installed and working @lauradurandv? This doesn't need to block the PR, but I'm still seeing some weird formatting - for example, I would expect prettier to insert a space here between `try` and `{`

Reply:
I do and I've played with settings to get it working, but I still do not see a difference, and I have not had time to look more into it.